solidity/contracts/deposit/DepositFunding.sol

```diff
@@ -169,7 +162,11 @@ library DepositFunding {
         bool _isFraud = _d.submitSignatureFraud(_v, _r, _s, _signedDigest, _preimage);
         require(_isFraud, "Signature is not fraudulent");
         _d.logFraudDuringSetup();
-        distributeSignerBondsToFunder(_d);
+
+        // allow deposit owner to withdraw sized bonds after contract termination;
```

Comment:
Seized? Guessing this is meant to be capitalized + end with a period as well <_<